modules/es-extensions/publisher/extensions/app/greg_publisher/pages/password.jag

```diff
@@ -0,0 +1,72 @@
+<%
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var caramel;
+require('/modules/publisher.js').exec(function (ctx) {
+    var log = new Log('Server_passwords');
```

Comment:
When defining a jaggery log instance, best practice is to provide the file path so that we can easily identify which jaggery file is printing the log. For example
var log = new Log ('site/pages/index.jg').

Otherwise, we will see a log saying it was printed by "Server_passwords" and we wont know where to look. Please fix this place and any other places.